frontend/gist-frontend/src/Components/CreateUserComponent.tsx

```diff
@@ -0,0 +1,142 @@
+import React from "react";
+import { Form, FormField, Button, Box, Layer, Heading, Select } from "grommet";
+import { User, UserFemale, UserManager } from "grommet-icons";
+
+interface ICreateUserComponentProps {}
+
+interface ICreateUserComponentState {
+  showPictureModal: boolean;
+  image: string;
+}
+
+export default class CreateUserComponent extends React.Component<
+  ICreateUserComponentProps,
+  ICreateUserComponentState
+> {
+  state = {
+    showPictureModal: false,
+    image: "User"
+  };
+  onImageButtonClick = () => {
```

Comment:
we should rename this to something more like select image, i kept looking around for an 'Image Button' like a button with an image and was confused

Reply:
Wasn't crazy about that name anyway; some weird names float around my head that early in the morning.